packages/ui-library/src/elements/tooltip/style.css

```diff
@@ -11,18 +11,22 @@
 			yst-text-white
 			yst-text-xs
 			yst-rounded-lg;
+			white-space: normal; /* Wrap the text in the tooltip */
+			width: max-content;
+			max-width: 448px; /* Set the maximum width of the tooltip. The value in pixels is equivalent to the maw-w-md class*/
```

Comment:
Why not use Tailwind classes here too?
Using Tailwind makes the actual values dependent on the configuration, which is a nice layer for the UI library.
Then the `px` value is not inline with the `rem` units that Tailwind uses: which makes it dependent on the font size of the root element. I.e. more flexibility.

Though the size is a bit arbitrary already. For example, in the story this max-width, combined with position left, makes the left part of the tooltip be hidden without the ability to scroll. To be fair, I wouldn't know how to solve this in an abstract way within this component. Just making sure you are aware.

Reply:
My bad 😅  you are totally right! At some point I was testing Tailwind css properties that didn't seem to be supported by our yst layer, and instead I tested it with native css as `text-wrap: wrap; `. Then I continued writing css 🙇‍♂️  Fair point 👍 

Regarding the `max-width` property, I agree it is a bit arbitrary. My proposal to deal with the tooltip to not be hidden in the story when placed on the left or on the right, would be to add a `max-width` property depending to the position of the tooltip, like: 

```
.yst-tooltip--top {
			@apply 
			yst-max-w-md
			...
			
.yst-tooltip--bottom {
			@apply 
			yst-max-w-md
			...

.yst-tooltip--right {
			@apply 
			yst-max-w-xs
             ...
             
.yst-tooltip--left {
			@apply 
			yst-max-w-sm
             ...
```
But, I believe it is easier to have just the property in the `yst-tooltip` root class, and adjust it accordingly depending on the user case. In any case I believe is to the @Yoast/ux-team to decide upon it.